templates/services/asynchronous_invocation/src/asynchronous_invocation/tests/main_test.bal

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Does it require to have a test without doing anything related to the implemented logic? Let's delete if it not necessary.

Reply:
Done